test/common/quic/envoy_quic_client_session_test.cc

```diff
@@ -163,6 +163,13 @@ class EnvoyQuicClientSessionTest : public testing::Test {
   QuicHttpClientConnectionImpl http_connection_;
 };
 
+TEST_F(EnvoyQuicClientSessionTest, QuicUnimplemented) {
```

Comment:
This looks like a bad merge. Do you need this change?

Reply:
i was trying to fix the test coverage...